p2p/CONFIG.md

```diff
@@ -0,0 +1,264 @@
+
+### Connectivity and Discovery
+
+The following set of configurations affects connectivity and discovery:
+
+| ENV                  | YAML                 | Default Value (SSV)    | Required |  Description                           |
+| ---                  | ---                  | ---                    | ---      | ---                                    |
+| `NETWORK_ID`         | `p2p.NetworkID`      | `ssv-testnet-v2`       | No       | ID of the network that the node runs on|
+| `BOOTNODES`          | `p2p.Bootnodes`      | Bootnode (Testnet)     | No       | ENR of network bootnodes               |
+| `SUBNETS`            | `p2p.Subnets`        | -                      | No       | Subnets that this node will join upon start, represented as hex string|
+| `P2P_MAX_PEERS`      | `p2p.MaxPeers`       | 60                     | No       | Maximum number connections to maintain |
+
+### Logging and Tracing
+
+The following set of configurations affects logging/tracing w.r.t network components.
+
+Note that in order to have a detail log, the node operator must set log level (`LOG_LEVEL`) to `debug`.
+
+| ENV                  | YAML                 | Default Value (SSV)    | Required |  Description                             |
+| ---                  | ---                  | ---                    | ---      | ---                                      |
+| `P2P_LOG`            | `p2p.P2pLog`         | false                  | No       | Flag to turn on/off network debug logs, and other traces within network layer  |
+| `PUBSUB_TRACE`       | `p2p.NetworkTrace`   | false                  | No       | Flag to turn on/off pubsub trace logs    |
+| `DISCOVERY_TRACE`    | `p2p.NetworkTrace`   | false                  | No       | Flag to turn on/off discovery trace logs |
+
+## Pubsub Parameters
+
+The following parameters are used when initializing and interaction with the pubsub router.
+
+**NOTE** that we compare default values with both libp2p defaults and ETH2.0 nodes. \
+For more information on libp2p defaults see 
+[gossipsub-v1.0 spec > parameters](https://github.com/libp2p/specs/blob/master/pubsub/gossipsub/gossipsub-v1.0.md#parameters). \
+For more information on ETH2.0 configuration please refer to 
+[their p2p spec](https://github.com/ethereum/consensus-specs/blob/dev/specs/phase0/p2p-interface.md#the-gossip-domain-gossipsub).
+
+### Last Seen
+
+Specifies how long a message will be remembered as seen.
+
+| Default Value (libp2p) | Default Value (SSV) | Default Value (ETH2) |
+| ---                    | ---                 | ---                  |
+| `2min`                 | `6.4min`            | `6.4min`             |
+
+Was increased to `6.4m` which is the duration of one epoch, as messages become redundant afterwards.
+
+### Outbound Queue Size
+
+The size of the queue that is used for outbound messages.
+
+| Default Value (libp2p) | Default Value (SSV) | Default Value (ETH2) |
```

Comment:
why the diff?

Reply:
mentions below, we use a power of 2 to be aligned with memory allocations